src/content/docs/style-guide/word-choice/usage-dictionary.mdx

```diff
@@ -374,6 +378,20 @@ For consistency across New Relic content, here are our general word usage guidel
     Unlike the Latin abbreviations [`e.g.`](#eg) or [`i.e.`](#ie)), you're welcome to use etc. It should have a period at the end (`etc.`). Ensure that you have several meaningful examples, though, before using it. For example, `cities including Portland, Seattle, Dublin, etc.` but not `cities including Portland, etc.`.
   </Collapser>
 
+    <Collapser
+    id="event"
+    title="event"
+  >
+To understand how to optimally write about events, we recommend reading: 
+* The [event section in the data types doc](/docs/data-apis/understand-data/new-relic-data-types/#event-data)
+* [The glossary entry for "event"](/docs/glossary/glossary/#event)
+
+Some language-related considerations when writing about events: 
+* Prior to the roll-out of our three other core data types of `Metric`, `Log`, and `Span`, at New Relic we referred to all NRDB-stored objects as "events." For this reason, internally and sometimes externally we still sometimes refer to all NRDB objects as "events." (For example, we might simply say "event data" when what we mean is NRDB-stored data of any type. Or, in [the query UI](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#inspected-count-limits), we refer to "events inspected" and this refers to all NRDB data types). So it's important to understand some of this ambiguity around how we use "event." For public resources, we should attempt to use "event" to refer to NRDB-stored data **that is not** `Metric`, `Log`, or `Span` data types. 
+* It's important to understand how "event" language can overlap with "metric" language. Both are broad general terms and due to that there can be some overlap in meaning. Because metrics are just measurements, all New Relic events can be said to be or contain metrics. For example, the events reported by our infrastructure monitoring contain metrics (here's [an integration doc listing the metrics attached to a specific event](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-monitoring-integration/#find-data)). 
```

Comment:
Can we knead this out? This overlap actually came up in my FSO workshop training. There was a debate about how the presenter was using metric. My first impulse is wanting an example of they're differentiated to complement the example about how they overlap.

Reply:
@zuluecho9, what are public ressources?